src/helpers/connectors.ts

```diff
@@ -9,9 +9,9 @@ const connectors = {
     network: '1',
     options: {
       projectId: 'e6454bd61aba40b786e866a69bd4c5c6',
-      chains: [1],
+      chains: [1, 10],
```

Comment:
Unfortunately this doesn't solve the proper configuration to work with Smart Contract Wallets, as this config is equal to enforcing the wallet to support both chainid 1 and 10 before an initial connection. More background on walletconnect docs: 

- https://docs.walletconnect.com/advanced/providers/ethereum#initialization
- https://walletconnect.com/blog/caip-25-implementation-guidance-migrating-to-empty-undefined-required-namespaces

The proper configuration is indeed by removing required chains filed, or by setting it to an empty array. I ran your code from source and I tested setting `chainId: []` with the following wallets. So far, it working with: 
 
✅ Trust Wallet (mobile)
✅ MetaMask (mobile)
✅ Rainbow (mobile)
✅  Valora Wallet
✅  Safe Wallet (mobile)
✅  Candide Wallet

Reply:
More wallets tested: 

✅ Zerion (mobile)
✅ Zengo mobile
✅ OKX (mobile)
✅ Uniswap wallet (mobile)

Reply:
Can confirm that setting required chains to an empty array works for Safe Wallet (web).

I am however getting an error when trying to vote for a proposal: `The method eth_signTypedData_v4 does not exist/is not available`. Moving `eth_signTypedData_v4` to `optionalMethods` resolves this issue as well.

Reply:
What do you think about setting the required chain depending on the network of the space that the user is currently on and only fall back to 1 if they are not on a specific space?

Reply:
That's also a possibility. Users aware of this will be able to connect, but the downgrade will be on the front page of Snapshot where there's no expected network. Snapshot will need to add a network selection bar in that case. 

Uniswap follows this configuration on their interface

Reply:
@ChaituVR thoughts? I don't want to leave this trailing behind. Let's get it right on snapshot and continue to work with the other wallets to have it sorted properly.